lib/tasks/svg_sprites.rake

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+task "svg_sprites:refetch" => [:environment] do |_, args|
```

Comment:
I'm not sure we need this rake task... are you thinking of a particular scenario where this would be useful?